dusty/http_server.py

```diff
@@ -0,0 +1,76 @@
+from uuid import uuid1
+from datetime import datetime
+from collections import namedtuple
+import calendar
+import logging
+
+from flask import Flask, request, jsonify
+
+from .compiler.spec_assembler import get_assembled_specs
+from .systems.docker import get_dusty_containers, get_docker_client
+
+logging.getLogger('werkzeug').setLevel(logging.ERROR)
```

Comment:
otherwise it logs every request at INFO level which gets really annoying since the 502 JS polls every second